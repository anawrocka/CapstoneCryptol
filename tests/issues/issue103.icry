:check (undefined : Bit)
let f x = [True] @ 1
:check (f : () -> Bit)