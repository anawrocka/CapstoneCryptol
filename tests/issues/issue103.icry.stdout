Loading module Cryptol

Run-time error: undefined
Using exhaustive testing.
ERROR for the following inputs:
()
invalid sequence index: 1
